TACC/Core-CMS
# Task/TUP-702 Make new windows accessible

## Overview

Creates accessibility for external links

## Related

- [TUP-702](https://tacc-main.atlassian.net/browse/TUP-702)

## Changes

- New windows are not inherently accessible
- We dynamically set the attribute of `target="_blank"`, so let's add `aria-description='Opens in new window.'`

## Testing

1. Add an external link
2. Make sure link has `aria-description` attribute and `rel=noopener`

## UI

<img width="2548" alt="Screenshot 2024-02-19 at 4 37 23 PM" src="https://github.com/TACC/Core-CMS/assets/63771558/edf63bc0-067a-4c9b-8cda-28d86bd996c3">
